code/modules/reagents/chemistry/reagents/toxins.dm

```diff
@@ -347,15 +347,15 @@
 				else
 					var/melted_something = FALSE
 					if(H.wear_mask && !(H.wear_mask.resistance_flags & ACID_PROOF))
-						qdel(H.wear_mask)
+						to_chat(H, "<span class='danger'>Your [H.wear_mask.name] melts away!</span>")
+						QDEL_NULL(H.wear_mask)
 						H.update_inv_wear_mask()
```

Comment:
```suggestion
						qdel(H.wear_mask)
```
The nulling will happen when the item gets deleted. It will also call `unEquip` which does a bunch of things, including updating the mask (and more which should also happen)